c2cgeoportal/tests/functional/test_fulltextsearch.py

```diff
@@ -128,6 +148,12 @@ def tearDown():  # noqa
             FullTextSearch.label == "label5").delete()
         DBSession.query(FullTextSearch).filter(
             FullTextSearch.label == "label6").delete()
+        DBSession.query(FullTextSearch).filter(
+            FullTextSearch.label == "label7").delete()
+        DBSession.query(FullTextSearch).filter(
+            FullTextSearch.label == "label70").delete()
+        DBSession.query(FullTextSearch).filter(
+            FullTextSearch.label == "label71").delete()
```

Comment:
71
70
to be sure that they are also ordered :-)

Reply:
You're right :-)